README.md

```diff
@@ -45,160 +88,88 @@ To run a specific unittest:
-# Update to the latest version of a dependency. Note: Our integration tests will
-# verify that they can re-generate Godeps.json from scratch, which means that
-# it's important that you have the same set of tags in your local copy of any
-# repositories as the origin does. That means you can't use `go get -u`, you
-# must cd to the path and use `git remote update`, which fetches tags. Example:
-cd $GOPATH/src/github.com/cloudflare/cfssl
-git remote update
-git checkout master
-git pull origin master
-# Re-vendor the dependencies from scratch
-godep save ./...
-git commit Godeps/ vendor/
-```
+Boulder is custom built for Let's Encrypt and is intended only to support the
+Web PKI and the CA/Browser forum's baseline requirements. In our experience
+often Boulder is not the right fit for organizations that are evaluating it for
+production usage. In most cases a centrally managed PKI that doesn't require
+domain-authorization with ACME is a better choice. For this environment we
+recommend evaluating [cfssl](https://github.com/cloudflare/cfssl) or a project
+other than Boulder.
 
-NOTE: If you get "godep: no packages can be updated," there's a good chance
-you're trying to update a single package that belongs to a repo with other
-packages. For instance, `godep update golang.org/x/crypto/ocsp` will produce
-this error, because it's part of the `golang.org/x/crypto` repo, from which we
-also import the `pkcs12` package. Godep requires that all packages from the same
-repo be on the same version, so it can't update just one. The error message is
-not particularly helpful. See https://github.com/tools/godep/issues/164 for the
-issue dedicated to fixing it.
+We offer a brief [deployment and implementation
+guide](https://github.com/letsencrypt/boulder/wiki/Deployment-&-Implementation-Guide)
+that describes some of the required work and security considerations involved in
+using Boulder in a production environment. As-is the docker based Boulder development environment is **not suitable for
+production usage**. It uses private key material that is publicly available,
+exposes debug ports and is brittle to component failure.
 
-NOTE: Updating cfssl in particular is tricky, because cfssl vendors
-`github.com/google/certificate-transparency/...` and
-`golang.org/x/crypto/ocsp/...`, which we also vendor. In practice this means you
-need to check out those two dependencies to the same version cfssl uses
-(available in `vendor/manifest` in the cfssl repo). If you fail to do this,
-you will get conflicting types between our vendored version and the cfssl vendored version.
+While we are supportive of other organization's deploying Boulder in
+a production setting we prioritize support and development work that favors
+Let's Encrypt's mission. This means we may not be able to provide timely support
+or accept pull-requests that deviate significantly from our first line goals. We
+will try our best to engage with folks who have done their homework and need
```

Comment:
Instead of "folks who have done their homework," how about "if you've thoroughly evaluated the alternatives and Boulder is definitely the best fit, we're happy to answer questions to the best of our ability."